### PR TITLE
upgrade dependencies

### PR DESCRIPTION
**Pull request by sgratzl**
closes 

**prerequisites**:

- [x] branch is up-to-date with the branch to be merged with, i.e. develop
- [x] build is successful
- [x] code is cleaned up and formatted

### CODE REVIEW

**Review by thinkh — COMMENTED** [review]
I've tested this branch locally and couldn't finish `yarn run build` or `yarn run build:dev`.  Here are my steps:

1. Switch to this branch
2. Run `yarn install`
3. Run `yarn -v` -> `3.2.0`
4. Run `node -v` -> `v14.19.0`
6. Run `yarn run build` -> never terminates until I cancel it and does not log anything

Do you have an idea what could be the problem here? I see with netlify build that it should build in general. 

Beside this there is only a minor comment below.

**Comment by sgratzl** [comment]
re build: not sure what is going on. I had to update to the latest version (16.15) and then it worked. I guess that is also the reason why the CI works.

**Comment by thinkh** [comment]
I can confirm that the build works with Node 16. 

> nvm use 16
> Now using node v16.14.2 (npm v8.5.0)

I've added a comment to the development setup in the README.md.

File: package.json [hunk 1]
```diff
@@ -71,17 +71,18 @@
     "clean:compile": "rimraf build/src \"build/*.tsbuildinfo\" \"*.tsbuildinfo\" ",
     "compile": "tsc -b ./tsconfig.json",
     "compile:dev": "tsc -b ./tsconfig.dev.json",
-    "fix": "npm run eslint:fix && npm run prettier:write",
+    "fix": "yarn run eslint:fix && yarn run prettier:write && yarn run stylelint:fix",
```
Comment (thinkh) [inline package.json:74]: I just saw this change. Do we need to change the other npm scripts below also from `npm` to `yarn`?

Comment (sgratzl) [reply package.json]: fixed

**Review by thinkh — APPROVED** [review]
Thanks for the compatibility changes. It looks good to me now.

**Pull request merged**